airbyte-cdk/java/airbyte-cdk/s3-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/staging/StagingConsumerFactory.kt

```diff
@@ -144,7 +146,7 @@ private constructor(
                 writeConfigs,
                 typerDeduper
             ), // todo (cgardens) - wrapping the old close function to avoid more code churn.
-            OnCloseFunction { _, streamSyncSummaries ->
+            { _, streamSyncSummaries ->
```

Comment:
now that we're in kotlin (no checked exceptions)... can we kill try-catch here? (... or just use the function returned from `GeneralStagingFunctions.onCloseFunction` directly)

feels weird to build a lambda that effectively just wraps another lambda

Reply:
yeah let me see if i can unify. i have no clue what the java equivalent code was 😅

Reply:
Simplified it and yet another cgardens TODO gone 😄

Reply:
we should track the number of times the string `cgardens` shows up in our code as a vanity metric :P